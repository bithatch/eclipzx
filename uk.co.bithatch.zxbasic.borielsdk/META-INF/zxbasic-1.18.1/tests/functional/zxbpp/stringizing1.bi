

#define x(a, b) q(a##b)
#define q(a)  #a

x(a, b)
q(5)
