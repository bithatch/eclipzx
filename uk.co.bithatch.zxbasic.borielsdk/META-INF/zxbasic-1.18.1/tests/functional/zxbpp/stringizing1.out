#line 1 "stringizing1.bi"


#line 4 "stringizing1.bi"
#line 5 "stringizing1.bi"

"ab"
"5"
